src/ontology/ro-edit.owl

```diff
@@ -2628,7 +2630,6 @@ SubObjectPropertyOf(obo:RO_0002237 obo:RO_0002444)
 # Object Property: obo:RO_0002238 (has component participant)
 
 AnnotationAssertion(obo:IAO_0000115 obo:RO_0002238 "X 'has component participant' Y means X 'has participant' Y and there is a cardinality constraint that specifies the numbers of Ys.")
-AnnotationAssertion(obo:IAO_0000117 obo:RO_0002238 <https://orcid.org/0000-0001-9625-1899>)
```

Comment:
@wdduncan, all your ORCID entries were removed in the edit file. This explains why your instance is no longer in the `imports/orcidio_import.owl`. It only imports information for all the ORCIDs present in the edit file. Please, add again using the annotation `term:contributor`.

Reply:
@anitacaron I think this checks out now. Please confirm.

Reply:
You didn't add back all your contributions. Do you confirm you want to remove them? It's missing in RO:0002238 and RO:0017001.

Reply:
Thanks @anitacaron! I missed that. I added back my contributions. I also changed the attribution to `https://orcid.org/0000-0003-2620-0345` (Asiyah) to `dcterms:contributor`.

Reply:
Thanks @wdduncan!